src/app/static/src/tests/components/runWorkflow/runWorkflowProgress.test.ts

```diff
@@ -46,16 +46,23 @@ const workflowStatus1 = {
     }
   }
 
+const realSetTimeout = setTimeout;
 
 describe(`runWorkflowProgress`, () => {
     beforeEach(() => {
         mockAxios.reset();
+        jest.useFakeTimers();
```

Comment:
I think you'd be better restricting this to just that tests that need it - otherwise someone is bound to add a new test that uses `setTimeout()`... and end up wondering why it isn't working

Reply:
This, I could not get to work.

Reply:
Does this pattern not work?
```javascript
        const wrapper = getWrapper();

        const realSetTimeout = setTimeout;
        jest.useFakeTimers();

        realSetTimeout(() => {
            expect(wrapper.vm.$data.pollingTimer).toBe(null);
            expect(setInterval).toHaveBeenCalledTimes(0);
            expect(clearInterval).toHaveBeenCalledTimes(0);

            jest.useRealTimers();
            done();
        });
```
If you prefer you could still do `jest.useRealTimers();` in a global `afterEach()`.